apps/studio/src/components/Sidebar.tsx

```diff
@@ -133,12 +134,23 @@ export const Sidebar: FunctionComponent<SidebarProps> = () => {
         ))}
       </div>
       <div className="flex flex-col">
+        <Tooltip content='Start Tour' placement='right' hideOnClick={true}>
+          <button
+            title="Start Tour"  
+            className='flex text-gray-500 hover:text-white focus:outline-none border-box p-4'
+            type="button"  
+            onClick={() => (driverObj.drive())}
```

Comment:
Can we store the current tour step so that user doesn't have to go through everything again?

Like this `driverObj.drive(4); // Starts at step 4`

Reply:
Yes, we can store the state of steps in local storage.

But there might be one problem: what if a user comes for the first time, checks 2-3 steps, and then closes the tour because they don't want to continue? When they come back, the tour will start from where they left off, this might be bad experience for the user.

Currently, the tour is only going to run for the first-time visit to the studio; after that, it won't appear because we are storing whether the user has visited or not in local storage.

We have also provided a button for the user to restart the tour at any time.

So, should I proceed with your given approach but there might be some problem with user experience?

Reply:
What I meant is if user clicks the start tour button, in that case this can done I think.

Reply:
My bad, I thought of first time user part. 
Sure, we can implement this.

So, should I add this part when user restart to tour again?

Reply:
I think so. WDYT @Amzani @KhudaDad414

Reply:
@Shurtu-gal Is there anything to add more?

Reply:
Is this done ?

Reply:
No, not yet waiting for the reply of other two people : )
Should I go for it?

Reply:
I think you should 👍🏻

Reply:
Done.
Added feature to resume tour from last position.

[Added feature to resume tour from last position.](https://github.com/user-attachments/assets/017b7e40-1cdf-48ea-be85-29a122ce0fa8)

Thank you @Shurtu-gal for recommending this amazing feature : )